test/Profiler/coverage_num_threads.swift

```diff
@@ -0,0 +1,14 @@
+// RUN: %target-swift-frontend -profile-generate -profile-coverage-mapping -num-threads 0 -emit-ir %S/Inputs/coverage_num_threads1.swift | %FileCheck %s -check-prefix=SINGLE-SOURCE
+
+// SINGLE-SOURCE: __LLVM_COV,__llvm_covmap
```

Comment:
These section names are different on Windows. How about something like: `FileCheck --implicit-check-not=llvm_coverage_mapping`, `SINGLE-SOURCE: llvm_coverage_mapping`?

Reply:
Ah right windows, originally I was doing this but `llvm_coverage_mapping` actually appears multiple times in the successful case, updated the test to this but with `llvm_coverage_mapping =` which should be the same